chia/simulator/full_node_simulator.py

```diff
@@ -704,3 +723,14 @@ async def wait_for_wallets_synced(
                 if await self.wallets_are_synced(wallet_nodes=wallet_nodes):
                     break
                 await asyncio.sleep(backoff_time)
+
+    async def wait_for_self_synced(
+        self,
+        timeout: Optional[float] = 5,
```

Comment:
```suggestion
        timeout: float = 5,
```
`adjusted_timeout()` seems to take a `float`, not an `Optional[float]`

Reply:
Maybe your editor isn't handling `@overload` properly: https://github.com/Chia-Network/chia-blockchain/blob/main/chia/util/timing.py#L41